src/components/ListButton.tsx

```diff
@@ -22,21 +23,37 @@ const ButtonText = styled.Text`
   font-size: 18px;
   line-height: 22px;
   color: ${Colors.Black};
-  margin-left: 22px;
 `;
 
 const Icon = styled.View`
+  margin-left: -7px;
+  margin-right: 22px;
   color: ${Colors.DarkBlue};
 `;
 
+const Next = styled(NextCircle)<{rotated?: boolean}>`
+  margin-left: auto;
+  transform: ${props => props.rotated ? 'rotate(90deg)' : ''};
```

Comment:
```suggestion
  transform: ${props => props.rotated ? 'rotate(90deg)' : 'none'};
```
так лучше я думаю
а ещё можно сделать так, чтобы transform просто не было в стилях. типо такого:

```suggestion
  ${p => p.rotated && `transform: 'rotate(90deg)'`};
```